src/foundations/lists/example-summary-list.njk

```diff
@@ -0,0 +1,17 @@
+{% from "components/list/_macro.njk" import onsList %}
+{{
+    onsList({
+        "variants": "summary",
+        "itemsList": [
+            {
+                "text": 'What are the dates of the sales period you are reporting for?'
+            },
+            {
+                "text": 'For the period 1 January 2015 to 2 February 2017, what was the value of your total turnover, excluding VAT?'
+            },
+            {
+                "text": 'Please indicate the reasons for any changes in the total turnover'
+            }
+        ]
+    })
```

Comment:
Think we should update this example to be more like a list of items because thats what its used for in eQ. The example in the DAC issue is a list of shops for example

Reply:
shall I align it with the rest of the lists examples instead? :

```
{
                "text": 'birth certificate'
            },
            {
                "text": 'national insurance number'
            },
            {
                "text": 'passport'
            }
```

Reply:
Yeah looks good

Reply:
all done :)